server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/RestService.java

```diff
@@ -77,6 +79,7 @@
    private final int maxContentLength;
    private int compressionLevel;
    private List<CorsConfig> corsConfigList;
+   private Transport transport;
```

Comment:
the `Transport` is the "service" isn't needed anymore and can be removed.

Reply:
Done